README.md

```diff
@@ -18,6 +18,8 @@ distribution. The Red Hat distribution powers [openshift.io](https://openshift.i
 Red Hat modifications against the upstream Che include:
 
 * [fabric8-analytics Language Server](https://github.com/fabric8-analytics/fabric8-analytics-lsp-server)
+* a different set of Che stacks than upstream. They can contain specific Red Hat configurations such as providing 'oc' on command-line
```

Comment:
few lines above there is this sentence: "The Red Hat distribution powers [openshift.io](https://openshift.io) developers workspaces."
I think that it might not be worthy to add something generic then. But maybe something more specific?
I don't know well rh-che content. Do you know which specific parts are different to ensure compatibility with openshift.io?

Reply:
@apupier thanks for your contribution. I think details about osio specific workspace management could be added in a separate PR